pkgs/tools/system/lanzaboote/tool.nix

```
@@ -0,0 +1,50 @@
+{ systemd
+, makeWrapper
+, binutils-unwrapped
+, sbsigntool
+, rustPlatform
+, fetchFromGitHub
+, lib
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lanzaboote_tool";
```

```suggestion
  pname = "lanzaboote-tool";
```
having matching attr and pname is nice